### ISSUE TITLE
1st (zero) element is not highlited with this.aureliaGrid.gridService.highlightRow(index, delay, delay)

### ISSUE THREAD

**Issue by UtmostCreator** [ghiscoding/aurelia-slickgrid#527]
### Your Environment

| Software          | Version(s) |
| ----------------- | ---------- |
| Aurelia           |  2.0.2
| Aurelia-Slickgrid | 3.2.1
| TypeScript        | vanilla JS
| Operating System  | windows 10 20h2
| NPM/Node/Yarn     | 14.15.4

### Context
I want to highlit the 1st (0) row on click.
![image](https://user-images.githubusercontent.com/22347676/108222757-eb2bac00-70ed-11eb-8498-94cf5fea3f35.png)

### Expected Behavior
It must highlight the 1st (zero - 0) row as others

### Current Behavior
it does not highlight the 1st (zero - 0)

### Possible Solution
make sure that check accepts 
`index >= 0
`
### Code Sample
```
this.aureliaGrid.gridService.highlightRow(0, delay, delay)
this.aureliaGrid.gridService.highlightRow(1, delay, delay)
```

### Preconditions
you must have a table with at least 2 rows.

**Comment by ghiscoding** [comment]
hi sorry I didn't have time to try that out, that method is in the Slickgrid-Universal common/GridService and I think it's because of this line actually

https://github.com/ghiscoding/slickgrid-universal/blob/20564a3096948626beada698460b72374a18ca7c/packages/common/src/services/grid.service.ts#L315-L317

probably have to change to `if (item && item[idPropName] !== undefined)`

**Comment by UtmostCreator** [comment]
Good day!

Yes, it should be sufficient to check like this, I believe 

you can also check it like this 

`if (item && typeof item[idPropName] !== 'undefined')`

Thank you!

**Comment by ghiscoding** [comment]
Closing this is now fixed and released under latest [version 3.4.0](https://github.com/ghiscoding/aurelia-slickgrid/releases), make sure to update both Aurelia-Slickgrid to `3.4.0` and all Slickgrid-Universal packages to `0.11.2`

Cheers ⭐
